browser/resources/brave_new_tab_page_refresh/BUILD.gn

```diff
@@ -14,6 +14,10 @@ transpile_web_ui("resources") {
       ] ]
   resource_name = "brave_new_tab_page_refresh"
   output_module = true
+  data = [
```

Comment:
I'd say such declaration should be enforced and asserted inside `transpile_web_ui` during the build. Can you compare the generated data_deps with the deps listed here? This will ensure any misconfiguration is caught and is always reflected properly in GN.

I'd also name this variable more specific to somehow reflect that it's required for webpack actions where we don't list all sources explicitly.

Reply:
I see you wrote it in "future improvements", but this feels more like a requirement tbh.

Reply:
`webpack_deps` maybe to be really explicit.

Reply:
deps by convention usually takes gn targets?
How about webSourceRoots?

in case we don't find a better name I'm tempted to just call it data since that is the gn nomenclature

Reply:
Personally I'm in favor of `data` :smile: `source_dirs` is also good, and I think other gn targets use `inputs`